src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
@Sami32 The same as above, stop thinking about them only from the point of view of the log, ```toString()``` need to generate a string representation of the object. As long as this information is stored double, it should be logged double.